.github/workflows/elm-to-gh-pages.yml

```diff
@@ -35,15 +35,15 @@ jobs:
         uses: jorelali/setup-elm@v5
         with:
           elm-version: 0.19.1
-      - run: elm make src/Main.elm
+      - run: elm make src/Main.elm --output=static/main.js
 
       - name: Setup Pages
         uses: actions/configure-pages@v3
 
       - name: Upload artifact
         uses: actions/upload-pages-artifact@v1
         with:
-          path: '.'
```

Comment:
if we keep `.` and we updated all realted places (`--outpu=main.js`), ... are we able to output html and js at root level? this would help the github pages deploy (see comment in main PR thread)

Reply:
The main issue I see is that the *index.html* file shall be uploaded as well. If the actions uploads the artifacts only (unsure how those are found), then it will not work either way. I'll look into it.

Reply:
I see the action works (primarily?) with tar files. We might pack the `static` directory into a .tar file and use the action for that.

Reply:
Ok, I tested on my branch. My original workflow was correct: the path directory will be tar-ed, uploaded and unpacked in the root.